automated-comments/rust/reverse-string/suggest_doing_bonus_test.md

```diff
@@ -0,0 +1,2 @@
+Consider expanding the solution to pass the feature-gated `test_grapheme_clusters` test case.
+The potential benefits of the expansion is the increased learning in the non-ASCII symbols handling area and the additional training in using the external crates, if the need arises to use one.
```

Comment:
This is _very_ wordy. Perhaps something simpler?

```suggestion
Understanding how non-ASCII symbols can be handled using external crates has great benefits down the line.
```

Reply:
Yes, you wording is much better.
Just a small moment - there are two items in the comment:

Understanding how non-ASCII symbols can be handled **and**  the usage of the external crates has great benefits down the line.

The student may try to reverse non-ASCII by hand - although daunting, but still a valid approach.

Reply:
> The student may try to reverse non-ASCII by hand - although daunting, but still a valid approach.

 Hehe, but they probably _shouldn't_ right?

```suggestion
Understanding how non-ASCII symbols can be handled and how to use external crates has great benefits down the line.
```

Like this?

Reply:
Yes, lets leave it at that